### PR TITLE
feat: Site Favicon Logo

### PR DESCRIPTION
**Pull request by refcell**
**Description**

Adds a configurable favicon logo to the fork-diff site generation.

Can be specified in the `Page` (top-level yaml section) as a string under the `logo` key.

An example logo is added with two gophers "spidermaning" (1) each other.

_1: Spidermaning, the spiderman meme looking at each other_

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Example:
 
 ```yaml
 title: "protolambda's Greeter fork"  # Define the HTML page title
+logo: "./logo.png"
```
Comment (protolambda) [inline README.md]: Would omit the dot here, and make it absolute: it's the img link as it appears in the html, not a linux filepath

Comment (refcell) [reply README.md]: Fixed :)

File: page.gohtml [hunk 2]
```diff
@@ -11,6 +11,8 @@
 
     <title>{{.Title}}</title>
 
+    <link rel="icon" type="image/x-icon" href={{.Logo}}>
```
Comment (protolambda) [inline page.gohtml]: Didn't notice this earlier, but looks like the quotes on the attribute value are missing after this generates?

Comment (refcell) [reply page.gohtml]: Fixed - good catch!

**Review by protolambda — APPROVED** [review]

**Pull request merged**